pages/linux/aa-update-browser.md

```diff
@@ -0,0 +1,26 @@
+# aa-update-browser
+
+> Update AppArmor browser profiles to use supported abstractions.
+> Part of the AppArmor suite.
+> Requires root privileges.
+> More information: <https://manpages.ubuntu.com/manpages/questing/en/man8/aa-update-browser.8.html>.
+
+- List available browser abstraction profiles:
+
+`sudo aa-update-browser -l`
+
+- Show what changes would be made to a profile without applying them (dry-run):
```

Comment:
```suggestion
- Show what changes would be made to a profile without applying them ([d]ry-run):
```
This will help users understand what the d is short for.